build.wake

```diff
@@ -11,17 +11,21 @@ global def hardfloatScalaModule =
   | setScalaModuleScalacOptions ("-Xsource:2.11", Nil)
   | addChisel3CompilerPlugin
 
-global def rocketchipMacros =
-  makeScalaModuleFromJSON here "rocketchipMacros"
-  | setScalaModuleRootDir "{rocketChipRoot}/macros"
-  | addMacrosParadiseCompilerPlugin
+def diplomacyRoot = findGitRepositoryWithFallback "diplomacy"
+
+global def diplomacyMacrosModule =
+  makeScalaModuleFromJSON here "diplomacyMacros"
+  | setScalaModuleRootDir "{diplomacyRoot}/diplomacy/macros"
+
+global def diplomacyModule =
+  def deps = diplomacyMacrosModule, Nil
+  makeScalaModuleFromJSON here "diplomacy"
```

Comment:
This is saying there's a key "diplomacy" in `ivydependencies.json`. You need to add one to get the tests using Wake to pass.

Reply:
Got it! Thank you, I think I used to add it, but seems to be dropped in a rebasing.